hw/ip/aes/model/aes_model.core

```diff
@@ -0,0 +1,26 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:model:aes:0.5"
+description: "AES Model"
+filesets:
+  files_cc:
```

Comment:
not sure it matters but should this be `files_dv `for the sake of VCS?

Reply:
It's just a name with no built-in meaning.

Reply:
I changed it to `files_dv`